### PR TITLE
SLA reports implementation

### PR DESCRIPTION
**Pull request by vikt0rs**

### CODE REVIEW

File: launchpad_reporting/criterias.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            "importance": bug.importance,
+            "with_customer_found": "with `customer-found` tag" if is_customer_found else "",
+            "threshold": threshold,
+        }
+        return template.format(**hint_data)
+
+
+class NonTriaged(BugCriteria):
+    """
+    Working out New bugs:
```
Comment (TimurNurlygayanov) [inline launchpad_reporting/criterias.py]: http://docs.openstack.org/developer/hacking/#docstrings

File: launchpad_reporting/criterias.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        return (bug.date_created < business_days_ago(self.threshold)
+                and (bug.milestone is None or bug.importance is None
+                     or bug.assignee is None)
+                )
+
+
+class SLAFullLifecycle(BugCriteria):
+    """
+     Recommended bug lifecycle from New to Fix Committed:
+
```
Comment (TimurNurlygayanov) [inline launchpad_reporting/criterias.py]: http://docs.openstack.org/developer/hacking/#docstrings

File: launchpad_reporting/config/sample.yaml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+      criterias:
+            - name: sla-non-triaged
+      # do not implement frequent-polling in current scope
+      frequent-polling: true
+        
+    - name: sla-report
+      text: SLA Report
+      parameter: milestone
+      group-by: team
+      display-triggere-criterias: true
```
Comment (dmitrymex) [inline launchpad_reporting/config/sample.yaml]: That is a typo here, the parameter should be
display-trigger-criterias

can you change that in config and in implementation itself?

Comment (vikt0rs) [inline launchpad_reporting/config/sample.yaml]: done

File: launchpad_reporting/criterias.py (hunk 1 above)
Comment (vikt0rs) [inline launchpad_reporting/criterias.py]: done

File: launchpad_reporting/criterias.py (hunk 2 above)
Comment (vikt0rs) [inline launchpad_reporting/criterias.py]: done

**Pull request merged**